frontend/www/js/omegaup/components/schools/Rank.vue

```diff
@@ -76,14 +99,21 @@ import CountryFlag from '../CountryFlag.vue';
   },
 })
 export default class Rank extends Vue {
+  @Prop() page!: number;
+  @Prop() length!: number;
+  @Prop() isIndex!: boolean;
```

Comment:
```suggestion
  @Prop() showHeader!: boolean;
```

en vez de decidir si es o no el índice, es mejor llamarlo por el efecto que tiene la bandera.